hazelcast/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxySyncListenerCompleter.java

```diff
@@ -58,10 +58,7 @@ protected void awaitLatch(CountDownLatch countDownLatch, ICompletableFuture futu
             while (currentTimeoutMs > 0
                     && !countDownLatch.await(COMPLETION_LATCH_WAIT_TIME_STEP, TimeUnit.MILLISECONDS)) {
                 if (future != null && future.isDone()) {
-                    Object response = future.get();
-                    if (response instanceof Throwable) {
-                        return;
-                    }
+                    future.get();
```

Comment:
Looks like all exceptions were silently ignored before, not sure is it an intended change or not.

Reply:
Actually the future at this point is a `ClientInvocationFuture`, a `CompletableFuture<ClientMessage>`.  The result of `future.get()` cannot be a `Throwable` here; if the future is completed exceptionally, even with previous version of code `future.get()` would throw an `ExecutionException` wrapping the throwable, so the `response instanceof Throwable` code path seems like dead code.